### DISCUSSION TITLE
Toward Including the James Webb Space Telescope among the Spacecrafts

### DISCUSSION THREAD

**Discussion by chanonr** [ivoyager/forum#14] [General]
Although the International Space Station (ISS) and the Hubble Space Telescope (Hubble) have been included in Planetarium, I didn't see a selection for the James Webb Space Telescope (JWST).  Are there plans to include it in Planetarium?  (Curiously, I found what looks like an empty function related to Lagrange Points!)

**Comment by charliewhitfield** [comment]
I'd like to include it, but I couldn't find an open source 3D model anywhere. If you can find one, please post a link!

I may have started some coding on this. I can't remember. If I had the model I'll do what it takes to put it at Earth's L2.
